lib/auth/auth_with_roles.go

```diff
@@ -2740,7 +2740,7 @@ func (a *ServerWithRoles) GetUsers(ctx context.Context, withSecrets bool) ([]typ
 			}); err != nil {
 				log.WithError(err).Warn("Failed to emit local login failure event.")
 			}
-			return nil, trace.AccessDenied("this request can be only executed by an admin")
+			return nil, trace.AccessDenied("User secret information requires admin rights and is not available from remote access")
```

Comment:
nit: rephrase so we still follow https://go.dev/wiki/CodeReviewComments#error-strings:

```suggestion
			return nil, trace.AccessDenied("reading User secrets requires admin rights and is not available from remote access")
```

I'll add the same comment to others, feel free to take none or all.